mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.purgeOrder = async (req, res) => {
+  try {
+    const result = await Order.findOneAndRemove({ _id: req.params.orderId });
+    if (!result) {
+      return res.status(404).json({ message: "Order not located" });
+    }
+    res.status(200).json({ message: "Order eradicated successfully" });
+  } catch (err) {
+    res.status(500).json({ message: errorHandler(err) });
```

Comment:
The error handling mechanism should be consistent throughout the codebase. In this case, the error response format is different from the other error handling instances in the code. For example, in the `orderById` function, the error response format is `{ error: errorHandler(err) }`, whereas here it's `{ message: errorHandler(err) }`. It should be consistent to maintain uniformity in error handling.